components/brave_webtorrent/stories/components/torrentFileList.tsx

```diff
@@ -0,0 +1,109 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react'
+import { Heading, Table } from 'brave-ui/components'
+import { Cell, Row } from 'brave-ui/components/dataTables/table/index'
+import { LoaderIcon } from 'brave-ui/components/icons'
+import * as prettierBytes from 'prettier-bytes'
+
+// Constants
+import { File, TorrentObj } from '../../extension/constants/webtorrentState'
+
+// Locale
+import locale from '../fake-locale';
+
+interface Props {
+  torrentId: string
+  torrent?: TorrentObj
+}
+
+export default class TorrentFileList extends React.PureComponent<Props, {}> {
```

Comment:
Are these components new or copied?

What would be great is to have TorrentFileList be a standalone dumb component, and have both the storybook and the actual extension page use it. Is that your plan?

Reply:
If I'm honest, my plan was to just get some things rendering and I was following the Welcome Experience storybook setup where it seemed like components were duplicated for Storybook presentation. In this case, I don't even think torrentFileList is rendering or being used at the moment. Just the header and footer but I was hoping clicking the "Start download" button would render this with fake data in a followup PR. So it could prob be removed for now in place of a better approach. This is all an amateur effort so this really helps.

Reply:
The existing storybooks are just copied over from brave-ui where they were duplicated somewhat by neccessity, so I wouldn't look to them for best practice. One of the aims of the Storybook is seeing what the current state of the components looks and interacts like, so having them duplicated would leave us with little confidence that they are up-to-date.
This is where containers / components come in to play. The actual extension would have app containers which provide props to the dumb components containing data and functions from the actual API / state. The storybook would have a container which provide props to the dumb components containing dummy data and functions. As many of the UI components as possible would be independent until ideally the whole UI could be rendered in this dummy data environment. But, piecemeal is fine too - starting with individual components, as long as they're not duplicated.

Reply:
Way smarter. Gonna take a stab at that and i'll DM with hangups.